# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/moz.build
# Commit: f357b2287591
# Full Hash: f357b2287591dd069c8ecdf5091c6a74ca173506
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 779684
# File Overlap Count: 1
# Description:
#   Bug 779684 - part 2: Get rid of `nsFrameTraversal` and `nsIFrameTraversal` r=emilio
#   
#   The class is used only for creating `nsFrameIterator`, but it's unnecessary
#   so that we can get rid of it and its interface.
#   
# ==============================================================================

diff -r 738fe6bbf913 -r f357b2287591 layout/base/moz.build
--- a/layout/base/moz.build	Sat Dec 23 05:39:38 2023 +0000
+++ b/layout/base/moz.build	Sat Dec 23 05:39:38 2023 +0000
@@ -48,7 +48,6 @@
     "nsFrameManager.h",
     "nsFrameTraversal.h",
     "nsGenConList.h",
-    "nsIFrameTraversal.h",
     "nsIPercentBSizeObserver.h",
     "nsIReflowCallback.h",
     "nsLayoutUtils.h",